### PR TITLE
Fix firefox fileupload

### PR DESCRIPTION
**Pull request by bierik**
Fixes https://github.com/4teamwork/ftw.file/issues/43

Drag'n'Drop is not working with FF (Overlay is not displayed)
When dragging a file with FF the datatransfer object looks like this:
`[ "application/x-moz-file", "Files" ]`. The test if a file exists does only inspect
the first index in the array. So look if the array contains!! the `Files` String.

### CODE REVIEW

File: ftw/file/resources/uploader.js [hunk 1]
```diff
@@ -58,7 +58,8 @@ var uploader = (function($) {
       unbindEvents();
       if (tests.dnd) {
         $(document).on('dragenter', function(event) {
-          if(event.dataTransfer.types[0] === 'Files') {
+          console.log(event.dataTransfer.types);
```
Comment (mbaechtold) [inline ftw/file/resources/uploader.js]: Did you miss to remove this line or is this by intention?

**Comment by bierik** [comment]
@mbaechtold updated

**Comment by mbaechtold** [comment]
It's working in Firefox 42 now but the green confirmation sign is slightly off the center:

screenrecording

**Comment by bierik** [comment]
@mbaechtold updated

**Comment by mbaechtold** [comment]
Cool, thank you @bierik :tada:

**Comment by mbaechtold** [comment]
Oh, I only noticed now that a changeling entry is missing. Could you please add on, @bierik.

**Comment by bierik** [comment]
@mbaechtold thx for the hint... changelog entry added

File: docs/HISTORY.txt [hunk 2]
```diff
@@ -5,6 +5,8 @@ Changelog
 1.11.3 (unreleased)
 -------------------
 
+- Fix drag'n'drop for files in Firefox.
+
```
Comment (mbaechtold) [inline docs/HISTORY.txt]: Please add you name :wink:

**Pull request merged**